pkg/vpcmodel/nodesExplainability.go

```diff
@@ -449,6 +449,7 @@ func (c *VPCConfig) processInput(srcName, dstName string) (srcNodes, dstNodes []
 	if err != nil {
 		return nil, nil, err
 	}
+	// TODO: next 'if' seems to be redundant now
```

Comment:
I assume this comment can be deleted if we do remove the errors returned from IPBlock generation earlier, and then checking that `len(srcNodes)` is not redundant, right?

Reply:
This is part was rewritten in my now merged PR. This code is no longer relevant